drivers/mpu9x50/Kconfig

```diff
@@ -5,30 +5,44 @@
 # directory for more details.
 #
 
-if TEST_KCONFIG
 
-choice
-    bool "MPU-9X50 accelerometer/magnetometer/gyroscope"
-    optional
+menuconfig MODULE_MPU9X50
```

Comment:
I think you are missing the `CONFIG_MODULE_MPU9X50=y` in the `app.config` of the test application.

Reply:
Hmm I wonder if we want it like that or for the entry point to be `MODULE_MPU9150` only?